tox.ini

```
@@ -2,7 +2,8 @@
 envlist = py35-django22,py38-django{22,30}
 
 [doc8]
-max-line-length = 120
+; D001 = Line too long. Reason: No hard line-breaks in docs.
```

```suggestion
; D001 = Line too long. Reason: No hard line-breaks in docs (see OEP-0019).
```

May as well link the OEP.